### PR TITLE
New Module: na_ontap_snapshot_policy.py

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
New Module: na_ontap_snapshot_policy -- Create and delete policys
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
- na_ontap_snapshot_policy.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by lonico — APPROVED** [review]
shipit

File: test/units/modules/storage/netapp/test_na_ontap_snapshot_policy.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+from units.compat.mock import patch, Mock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible.module_utils.netapp as netapp_utils
+
+from ansible.modules.storage.netapp.na_ontap_snapshot_policy \
+    import NetAppOntapSnapshotPolicy as my_module
+
+if not netapp_utils.has_netapp_lib():
+    pytestmark = pytest.skip('skipping as missing required netapp_lib')
```
Comment (lonico) [inline test/units/modules/storage/netapp/test_na_ontap_snapshot_policy.py]: pytest.mark.skip

Though I'm surprised we did not see any bot error.

**Review by lonico — CHANGES_REQUESTED** [review]
We need to change pytest.skip to pytest.mark.skip

**Review by lonico — APPROVED** [review]

**Comment by lonico** [comment]
shipit

**Review by thedoubl3j — APPROVED** [review]
Simple, documentation looks good. tests and CI are in good shape, lgtm.

**Comment by thedoubl3j** [comment]
cc @thaumos

**Pull request merged**